### PR TITLE
Update azuread.py

### PR DESCRIPTION
**Pull request by dhana0918**
using access_token if id_token is not present.

I am using this in my application for Office365 oauth, always id_token is None so changed this 
code, please review and merge it.

### CODE REVIEW

**Comment by omab** [comment]
@dhana0918, also, could you add an entry to the `CHANGELOG.md` file about the change?

File: social_core/backends/azuread.py [hunk 1]
```diff
@@ -77,7 +77,11 @@ def get_user_details(self, response):
 
     def user_data(self, access_token, *args, **kwargs):
         response = kwargs.get('response')
-        id_token = response.get('id_token')
+        if response and response.get('id_token'):
+            id_token = response.get('id_token')
+        else:
+            id_token = access_token
+        #id_token = response.get('id_token')
```
[the remark this answers is not in the document]
Comment (dhana0918) [reply social_core/backends/azuread.py]: Remove the commented line

**Comment by dhana0918** [comment]
Hi @omab , 
Remove the commented line, and added to CHANGELOG.md. 

Thanks

**Comment by omab** [comment]
Merged, thanks!

**Pull request merged**